templates/scenarios/js/non-sso-tab/.webappignore

```diff
@@ -0,0 +1,26 @@
+.webappignore
+.fx
+.deployment
+.localSettings
+.vscode
+*.js.map
+*.ts.map
+*.ts
+.git*
+.tsbuildinfo
+CHANGELOG.md
+readme.md
+local.settings.json
+test
+tsconfig.json
+.DS_Store
+teamsapp.yml
+teamsapp.*.yml
+/env/
+/script/
+/node_modules/.bin
+/node_modules/ts-node
+/node_modules/typescript
```

Comment:
Since it is js template, do we need to include some ts file in this ignore file?